bluemira/codes/process/mapping.py

```diff
@@ -156,7 +156,9 @@ class TFCoilConductorTechnology(Model):
     "tk_vv_in": ("d_vv_in", "m"),
     "tk_sol_ib": ("scrapli", "m"),
     "tk_sol_ob": ("scraplo", "m"),
-    "tk_ts": ("thshield", "m"),
+    "tk_ts": ("thshield_ob", "m"),
+    "tk_ts": ("thshield_vb", "m"),
+    "tk_ts": ("thshield_ib", "m"),
```

Comment:
This may not be the thermal shield as we understand it in BLUEMIRA... PROCESS has a VV which has a thermal shield on the inside of it or something similar.. Please check! Before we used to to tk_vv [BLUEMIRA] = tk_vv [PROCESS] + tk_ts [PROCESS].